.github/workflows/postman-prod.yml

```diff
@@ -4,6 +4,9 @@ on:
     push:
         branches:
             - main
+    pull_request:
+        branches:
+            - main
```

Comment:
This means the sync doesn't wait for the PR to be merged. Is that correct behavior?